tests/coordptr.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python2
 from __future__ import absolute_import, division
```

Comment:
Yes. I know. `futurize` does not combine automatically and I wonder whether it's better for them to be together or separate (and I think I prefer separate). But I have not gone through the old code to split older items.